oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -97,10 +97,10 @@ public class ServiceAccountCredentials extends GoogleCredentials
   private static final String PARSE_ERROR_PREFIX = "Error parsing token refresh response. ";
   private static final int TWELVE_HOURS_IN_SECONDS = 43200;
   private static final int DEFAULT_LIFETIME_IN_SECONDS = 3600;
-  private static final int DEFAULT_NUMBER_OF_RETRIES = 3;
   private static final int INITIAL_RETRY_INTERVAL_MILLIS = 1000;
   private static final double RETRY_RANDOMIZATION_FACTOR = 0.1;
   private static final double RETRY_MULTIPLIER = 2;
+  public static final int DEFAULT_NUMBER_OF_RETRIES = 3;
```

Comment:
Package-private? Otherwise this will need to be stable API.

Reply:
fixed